prow/jobs/test-infra/buildpack.yaml

```diff
@@ -300,6 +300,23 @@ prow_clonerefs_job_template: &prow_clonerefs_job_template
           memory: 1.5Gi
           cpu: 0.8
 
+kyma_cleaners_job_template: &kyma_tools_job_template
+  run_if_changed: "^development/tools/"
+  <<: *common_job_template
+  spec:
+    containers:
+      - image: eu.gcr.io/kyma-project/prow/test-infra/bootstrap:v20181121-f3ea5ce
+        securityContext:
```

Comment:
I'm not sure we need privileged here

Reply:
We need privileged mode to build docker image